test/test_package.py

```diff
@@ -164,6 +185,7 @@ def test_validate_package(self):
         self.assertRaises(InvalidPackage, Package.validate, pack)
         pack.authors = []
         pack.validate()
+
```

Comment:
I know we don't normally like random whitespace changes, but this is within the same function, and makes the test more consistent and readable.